eo-maven-plugin/src/test/java/org/eolang/maven/HomeTest.java

```diff
@@ -102,29 +106,58 @@ void existsInDirWithSpecialSymbolsTest(@TempDir final Path temp) throws IOExcept
         final String filename = "EOorg/EOeolang/EOmath/EOnan$EOas_int$EO@";
         final byte[] bytes = filename.getBytes("CP1252");
         final String decoded = new String(bytes, "CP1252");
-        new Home(Paths.get("directory")).save("any content", temp.resolve(decoded));
+        new Home(Paths.get("directory")).save("any content", Paths.get(decoded));
         MatcherAssert.assertThat(
-            new Home(Paths.get("directory")).exists(temp.resolve(filename)),
+            new Home(Paths.get("directory")).exists(Paths.get(filename)),
             Matchers.is(true)
         );
     }
 
     @Test
     void loadBytesFromExistingFile(@TempDir final Path temp) throws IOException {
-        final Home home = new Home();
-        final String filename = "foo";
+        final Home home = new Home(temp);
         final String content = "bar";
-        final Path subfolder = temp.resolve("subfolder").resolve(filename);
+        final Path subfolder = Paths.get("subfolder").resolve("foo.txt");
         home.save(content, subfolder);
-        final Bytes bytes = home.load(subfolder);
-        final TextOf text = new TextOf(bytes);
-        MatcherAssert.assertThat(text, Matchers.equalTo(new TextOf(content)));
+        MatcherAssert.assertThat(
+            new TextOf(home.load(subfolder)),
+            Matchers.equalTo(new TextOf(content))
+        );
     }
 
     @Test
     void loadFromAbsentFile(@TempDir final Path temp) {
+        Assertions.assertThrows(
+            NoSuchFileException.class,
+            () -> new Home(temp).load(temp.resolve("nonexistent"))
+        );
+    }
+
+    @Test
+    void savesToCwd() throws IOException {
         final Home home = new Home();
-        final Path absent = temp.resolve("nonexistent");
-        Assertions.assertThrows(NoSuchFileException.class, () -> home.load(absent));
+        final String content = "bar";
+        final Path subfolder = Paths.get("subfolder").resolve("foo.txt");
+        home.save(content, subfolder);
+        MatcherAssert.assertThat(
+            new TextOf(home.load(subfolder)),
+            Matchers.equalTo(new TextOf(content))
+        );
+    }
+
+    /**
+     * Write file content.
+     * @param target Path to a file
+     * @param content Content
+     * @throws IOException In case I/O exceptions.
+     */
+    private static void write(final Path target, final String content) throws IOException {
```

Comment:
Can we use `Home.save()` instead?

Reply:
@volodya-lombrozo No. `Home.save()` is not used here on purpose. For some tests we need to create files not via `Home` object functionality in order to have valid test case. `save()` and `load()` methods share some logic and if used in pair for test may hide bugs.

Reply:
@mximp I see, you right. It's better to use a `Java nio` then:
```java
Files.write(target, content.getBytes())
```
it looks shorter

Reply:
@volodya-lombrozo Thanks. Fixed.